### DISCUSSION TITLE
Cron / webhooks

### DISCUSSION THREAD

**Discussion by Andrey-Bazhanov** [keystonejs/keystone#6568] [Event Q&A Q3 2021]
Which is preferred way to implement scheduled tasks?

**Comment by gautamsi** [comment]
you can trigger the cron jobs using node-cron, you should do this in onConnect event where you have keystone system functional.

for webhooks you need to implement your own. possibly in after change

**Reply by bladey** [reply]
Thanks for the answer @gautamsi, @Andrey-Bazhanov we agree with this approach! Let us know if you have any further questions on the Keystone Community Slack - https://community.keystonejs.com
